apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import Cookies from 'js-cookie'
 import django from 'django'
 
-import ModerationNote from './ModerationNote'
+import { ModerationStatement } from './ModerationStatement'
```

Comment:
i guess this is the wording we use for this purpose right? `statement` not `feedback`?

Reply:
I got statement from @fuzzylogic2000 comment [here](https://taiga.liqd.net/project/liqd-liquid-software/us/5761?milestone=302) I thought statement was more descriptive then feedback, what do you guys think?
Actually it could actually be `ModerationStatementForm` or `ModerationFeedbackForm`

Reply:
Yes, as the model and the field are called statement, I think statement is good! 
```
class ModeratorCommentStatement(UserGeneratedContentModel):
    statement = RichTextField(
        blank=True,
        # Translators: kosmo
        verbose_name=_('Moderator feedback'),
    )
    comment = ...
```
(But the app is called moderatorfeedback as that already existed.)